utils/submission.js

```diff
@@ -73,6 +73,19 @@ const submit = async (res, data, uid) => {
         }
     }
 
+    if (data[399159511] || data[996038075]) { // update user profile F + L name in search query
+        const { retrieveUserProfile } = require('./firestore');
+        const previousQuery = (await retrieveUserProfile(uid))[0].query;
+        const firstName = data[399159511]?.trim()?.toLowerCase();
```

Comment:
This seems like a complicated way of doing things...

if(data[399159511]) data['query.firstName'] = data[399159511].trim().toLowerCase();
if(data[996038075]) data['query.lastName'] = data[996038075].trim().toLowerCase();

Reply:
your suggested way of code replaces the key-value pairs in query object instead of appending to the object. 
I think its concise